console/console-init/ui/src/modules/iot-device-detail/containers/DeviceInfoContainer/DeviceInfoContainer.tsx

```diff
@@ -36,13 +44,17 @@ export const DeviceInfoContainer: React.FC<IDeviceInfoContainerProps> = ({
     refetchQueries
   );
 
-  const { credentials, jsonData, viaGateway } = data?.devices?.devices[0] || {};
+  const { credentials, ext: extString, via } = data?.devices?.devices[0] || {};
   const credentialsJson = credentials && JSON.parse(credentials);
-  const deviceJson = jsonData && JSON.parse(jsonData);
+
+  const ext = extString && JSON.parse(extString);
+
+  const viaGateway = via && via?.length > 0 ? true : false;
```

Comment:
I think we don't need ternary operator here. 
  const viaGateway = via && via?.length > 0;
It will handle true and false